src/tests/ftest/telemetry/dfs_client_telemetry.yaml

```diff
@@ -0,0 +1,46 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
+
+timeout: 180
+
+server_config:
+  name: daos_server
+  engines_per_host: 1
+  engines:
+    0:
+      targets: 4
+      nr_xs_helpers: 0
+      storage:
+        0:
+          class: ram
+          scm_mount: /mnt/daos
+  system_ram_reserved: 1
+
+agent_config:
+  telemetry_port: 9191
+  telemetry_retain: 30s
+  telemetry_enabled: true
+
+pool:
+  scm_size: 2G
+
+container:
+  type: POSIX
+  control_method: daos
```

Comment:
FYI, this is now the default.

Reply:
The container type and control_method? Or just the control_method?